pkg/kv/kvserver/raftlog/entry.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package raftlog
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/kvserverbase"
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/kvserverpb"
+	"github.com/cockroachdb/cockroach/pkg/storage"
+	"github.com/cockroachdb/cockroach/pkg/storage/enginepb"
+	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
+	"github.com/cockroachdb/errors"
+	"go.etcd.io/etcd/raft/v3/raftpb"
+)
+
+// Entry contains data related to a raft log entry. This is the raftpb.Entry
+// itself but also all encapsulated data relevant for command application.
+type Entry struct {
+	Meta enginepb.MVCCMetadata // only set after LoadFromRawValue
```

Comment:
Can we get rid of this field? It doesn't appear to be used anywhere, it's unclear what purpose it serves, and it's unfortunate that it's sometimes uninitialized.